Header header
string host
string mac